templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,11 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 40**
+
+* [`PATCH /settings/notifications`](/api/update-notification-settings):
+  Add `enable_notification_on_unsubscribe_stream` setting .
```

Comment:
Add another line explaining the setting.

Reply:
Sure